doc/script_commands.txt

```diff
@@ -6231,6 +6231,15 @@ Example usage:
 
 ---------------------------------------
 
+*specialeffectnum(<effect number>, <num1>, <num2>{, <send_target>{, <unit id>{, <account id>}}})
+*specialeffectnum(<effect number>, <num1>, <num2>{, <send_target>{, "<NPC Name>"{, <account id>}}})
+
```

Comment:
Please explain how `<num1>` and `<num2>` should be used.

Reply:
it always effect depend. num1 and num2 combined into int64 and then sent to effect function

Reply:
Okay, I see...